python/lsst/obs/subaru/isr.py

```diff
@@ -247,6 +247,11 @@ def runDataRef(self, sensorRef):
                 self.saturationDetection(ccdExposure, amp)
             if self.config.doSuspect and not badAmp:
                 self.suspectDetection(ccdExposure, amp)
+            ampMask = afwImage.MaskU(ccdExposure.getMaskedImage().getMask(), amp.getRawDataBBox(),
+                                     afwImage.PARENT)
+            maskVal = ampMask.getPlaneBitMask([self.config.saturatedMaskName, self.config.suspectMaskName])
+            if numpy.all(ampMask.getArray() & maskVal > 0):
+                badAmp = True
             if self.config.doOverscan and not badAmp:
```

Comment:
The `> 0` converts the `ampMask.getArray() & maskVal` to a boolean array, which allows use of `numpy.all` to see if all the values are `True`.

I think the overscan subtraction fails because the array it's trying to fit with a spline is empty.

I haven't seen the case of a single unsaturated pixel, but I have seen failures when the amp is entirely saturated.  I suggest we can whack that mole if it pops up.